### DISCUSSION TITLE
How do I see annotations from other pdf readers? They only show up as yellow crosses on sioyek

### DISCUSSION THREAD

**Discussion by emilBeBri** [ahrm/sioyek#602] [Q&A]
as stated - I have a pdf with comments from a colleague, but on siyoek (and on zathura too) the annotiations are only showed as yellow crosses, and I can't see how to see them. 

In the documentation, the only command that shows up when searching for 'annotation' and 'comment is the command 'embed_annotion', which is not about this at all (but rather the reverse direction)

on manjaro with i3, sioyek version 2.0.0

**Comment by ahrm** [comment]
I assume they are annotations not supported by muPDF (the engine we use to render PDFs). It is not really something I could fix, you should contact muPDF developers.
